boards/common/esp8266/include/periph_conf_common.h

```diff
@@ -110,7 +110,7 @@ extern "C" {
 #define SPI_NUMOF   1                       /**< Number of SPI interfaces */
 #define SPI_DEV(x)  ((unsigned int)(x+1))   /**< SPI_DEV to SPI hardware mapping */
 
-#define SPI0_DEV         SPI_DEV(0) /**< HSPI / SPI_DEV(0) device */
+#define SPI0_CTRL        SPI_DEV(0) /**< HSPI / SPI_DEV(0) device */
```

Comment:
Unrelated? At least, this is not ESP32.

Reply:
Ups, global subst with sed :grimacing: I will remove it from the commit.